cypress/e2e/bookStore.cy.js

```diff
@@ -2,10 +2,35 @@
 
 describe('Book Store app', () => {
   before(() => {
-    
+
+    cy.viewport(1200, 1200)
+    cy.visit('https://demoqa.com/login')
   });
 
-  it('', () => {
-    
+  it('login, add to cart, delete from cart', () => {
```

Comment:
the name of your `it` should continue the `describe` name, so
```suggestion
  it('should allow to login, add a book to cart and delete it from cart', () => {
```

Reply:
fixed